src/components/StudentList.tsx

```diff
@@ -7,27 +8,33 @@ import {
   TableCell,
   TableRow,
   Paper,
+  Button,
+  IconButton,
+  SvgIcon,
 } from "@mui/material";
+type FontAwesomeSvgIconProps = {
+  icon: any;
+};
+
 function Home(
   Name: string,
   Marks: number,
   Subject: string,
   Grade: string,
-  Date: string,
-  Action: string
+  Date: string
 ) {
-  return { Name, Marks, Subject, Grade, Date, Action };
+  return { Name, Marks, Subject, Grade, Date };
 }
 
-const rows = [
-  Home("Ayaz", 95, "Calculus", "A", Date(), "abc"),
-  Home("Usama", 75, "Python", "B", Date(), "abc"),
-  Home("Ali", 91, "ITC", "A", Date(), "abc"),
-  Home("Salman", 76, "Data structure", "B", Date(), "abc"),
-  Home("Sameed", 90, "Entrepreneurship", "A", Date(), "abc"),
-];
+type Iprops = {
+  title: string;
+};
+
+export default function Student({ list, fetchStudentData }: any) {
```

Comment:
Component name should match file name. rename it to be StudentList or StudentTable.
I prefer StudentTable. Change the file name to StudentTable